test/chaat/db/user_test.clj

```diff
@@ -0,0 +1,71 @@
+(ns chaat.db.user-test
+  (:require [chaat.fixture :as fixture]
+            [clojure.test :refer :all]
+            [chaat.db.user :as db.user]
+            [chaat.model.user :as model.user]
+            [crypto.password.bcrypt :as bcrypt]
+            [java-time.api :as jt]))
+
+(use-fixtures :each fixture/test-fixture)
+
+(defn- time-stub
+  "Return a fixed timestamp"
+  []
+  (jt/instant "2023-06-13T10:07:03.172Z"))
```

Comment:
suggestion: create a variable with this time, and return that variable in this fn. This will make it easy for you to use this time value in your tests

```
(def stubbed-time (jt/instant "2023-06-13T10:07:03.172Z"))
(defn- time-stub [] stubbed-time)

.
.
in an assertion..
(is (= (:time {..} stubbed-time))
```

Reply:
Yes, this is much more convenient, thanks!